### ISSUE TITLE
[error message feedback] provide feedback

### ISSUE THREAD

**Issue by sraje** [MicrosoftDocs/feedback#2446]
**🛑 IMPORTANT**: You can get your feedback addressed faster if you **use the comment section for the article in which you encountered a problem**.

**Problem:**
What's wrong with the error message? For example,
- Is error message clear and actionable?
- Is there a documentation to provide more information?
- If there is a documentation, is it relavent and helpful?
- Is line number correct?
- etc.

**Log code info**
(This is auto filled, no need to modify)

| Severity   | Type   | Code   | Category   | File   | Message   |  Source  | Line   | Time   |
|------------|--------|--------|------------|--------|-----------|----------|--------|--------|
| Warning | User |  |  | portal-add-subtitles-captions.html | Can not find the relative path: media/stream-utils/cleanvtt.html in manifest. So skip to fix the invalid link. | OPS.Microsoft.OpenPublishing.Build.Applications.HtmlToPdfTool.exe | | 01/28/2020 22:24:53 |

**Link to report**
(This is auto filled, no need to modify)

[report url](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C28%5Cbf956125-6efb-026a-e097-0ec3099efd1d%5CCommit%5C202001282220396167-live%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TcSgbKLe3FU0tiv80lymixoMVrcqjz5xUESJQVdrs%2Fk%3D&st=2020-01-28T22%3A25%3A10Z&se=2020-02-28T22%3A30%3A10Z&sp=r)

**Comment by docs-product** [comment]
@sraje - is there a specific issue with the build?

**Comment by sraje** [comment]
Yes - it yielded a warning and I'm not really sure what the issue is or how to fix it.
